install.sh

```diff
@@ -1016,7 +1016,7 @@ if [[ "$(which brew)" != "${HOMEBREW_PREFIX}/bin/brew" ]]
 then
   cat <<EOS
 - Run these two commands in your terminal to add Homebrew to your ${tty_bold}PATH${tty_reset}:
-    echo 'eval "\$(${HOMEBREW_PREFIX}/bin/brew shellenv)"' >> ${shell_profile}
+    echo -e '\\neval "\$(${HOMEBREW_PREFIX}/bin/brew shellenv)"' >> ${shell_profile}
```

Comment:
The extra `\`s make this a bit inscrutable. Instead, I suggest starting with a comment to guarantee that the `eval` call starts at a new line.

```suggestion
    echo '# Set PATH, MANPATH, etc., for Homebrew.' >> ${shell_profile}
    echo 'eval "\$(${HOMEBREW_PREFIX}/bin/brew shellenv)"' >> ${shell_profile}
```

Same for the second change below.

Reply:
OK.